packages/config/src/Config.ts

```diff
@@ -0,0 +1,89 @@
+import * as enums from './enums';
+import { Autocomplete } from './Autocomplete'
+import { Recommendation } from './Recommendation'
+import { Search } from './Search'
+import { Content } from './Content'
+import { Method } from '@findify/sdk/types/request'
+
+export type Config = {
+  /** Merchant API key */
+  key: string
+
+  /** Merchant ID, required to connect Chrome DevTools */
+  merchant_id: number
```

Comment:
@Bitaru let's make this field name in camel case as well

Reply:
it will require Devtools update -_-